core/src/com/unciv/logic/automation/civilization/MotivationToAttackAutomation.kt

```diff
@@ -160,95 +237,6 @@ object MotivationToAttackAutomation {
-        // We might try to divide the current production by the getUnitSupplyProductionPenalty()
-        // but it only is true for our turn and not the previous turn and might result in odd values
-
-        val productionRatio = civInfo.getStatForRanking(RankingType.Production).toFloat() / otherCiv.getStatForRanking(RankingType.Production).toFloat()
-        val productionRatioModifier = when {
-            productionRatio > 2f -> 15
-            productionRatio > 1.5f -> 7
-            productionRatio > 1.2 -> 3
-            productionRatio > .8f -> 0
-            productionRatio > .5f -> -5
-            productionRatio > .25f -> -10
-            else -> -10
-        }
-        return productionRatioModifier
-    }
-
-    private fun getScoreRatioModifier(otherCiv: Civilization, civInfo: Civilization): Int {
-        // Civs with more score are more threatening to our victory
-        // Bias towards attacking civs with a high score and low military
-        // Bias against attacking civs with a low score and a high military
-        // Designed to mitigate AIs declaring war on weaker civs instead of their rivals
-        val scoreRatio = otherCiv.getStatForRanking(RankingType.Score).toFloat() / civInfo.getStatForRanking(RankingType.Score).toFloat()
-        val scoreRatioModifier = when {
-            scoreRatio > 2f -> 15
-            scoreRatio > 1.5f -> 10
-            scoreRatio > 1.25f -> 5
-            scoreRatio > 1f -> 0
-            scoreRatio > .5f -> -2
-            scoreRatio > .25f -> -5
-            else -> -10
-        }
-        return scoreRatioModifier
-    }
-
-    private fun getDefensivePactAlliesScore(otherCiv: Civilization, civInfo: Civilization, baseForce: Float, ourCombatStrength: Float): Int {
-        var theirAlliesValue = 0
-        for (thirdCiv in otherCiv.diplomacy.values.filter { it.hasFlag(DiplomacyFlags.DefensivePact) && it.otherCiv() != civInfo }) {
-            val thirdCivCombatStrengthRatio = otherCiv.getStatForRanking(RankingType.Force).toFloat() + baseForce / ourCombatStrength
-            theirAlliesValue += when {
-                thirdCivCombatStrengthRatio > 5 -> -15
-                thirdCivCombatStrengthRatio > 2.5 -> -10
-                thirdCivCombatStrengthRatio > 2 -> -8
-                thirdCivCombatStrengthRatio > 1.5 -> -5
-                thirdCivCombatStrengthRatio > .8 -> -2
-                else -> 0
-            }
-        }
-        return theirAlliesValue
-    }
-
-    private fun getCombatStrengthModifier(ourCombatStrength: Float, theirCombatStrength: Float): Int {
-        val combatStrengthRatio = ourCombatStrength / theirCombatStrength
-        val combatStrengthModifier = when {
-            combatStrengthRatio > 3f -> 30
-            combatStrengthRatio > 2.5f -> 25
-            combatStrengthRatio > 2f -> 20
-            combatStrengthRatio > 1.5f -> 10
-            else -> 0
-        }
-        return combatStrengthModifier
-    }
```

Comment:
These all look like things we should be taking into account?

Reply:
They are taken into account, here: [powerAdvantageScore](https://github.com/MioBestWaifu/Unciv/blob/449c7bb43d16c7c07f545778c5e18114febc0bf6/core/src/com/unciv/logic/automation/civilization/MotivationToAttackAutomation.kt#L152). Except for defensive pact, forgot about it, will fix that

Reply:
OK, fair enough